build.gradle

```diff
@@ -198,11 +198,7 @@ dependencies {
   compile group: 'uk.gov.hmcts.reform', name: 'java-logging-spring', version: reformsJavaLoggingVersion
   compile group: 'uk.gov.hmcts.reform', name: 'java-logging-httpcomponents', version: reformsJavaLoggingVersion
   compile group: 'uk.gov.hmcts.reform', name: 'java-logging-appinsights', version: reformsJavaLoggingVersion
-
-  compile group: 'uk.gov.hmcts.reform', name: 'http-proxy-spring-boot-autoconfigure', version: '1.1.0'
-  compile group: 'uk.gov.hmcts.auth.checker', name: 'auth-checker-spring', version: '1.0.1'
-  compile group: 'uk.gov.hmcts.auth.provider.service', name: 'service-token-generator-spring', version: '1.0.5'
-  compile group: 'uk.gov.hmcts.auth.totp', name: 'totp', version: '1.0.1'
+  compile group: 'uk.gov.hmcts.reform.auth', name: 'auth-checker-lib', version: '2.1.1'
```

Comment:
because of your changes you can remove this line  maven { url "http://artifactory.reform.hmcts.net/artifactory/libs-release" }

Reply:
This has now been done.